egs/avspeech/local/loader/constants/__init__.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+STORAGE_DIR = os.environ.get("STORAGE_DIR", "../../storage_dir")
```

Comment:
```suggestion
STORAGE_DIR = os.environ.get("STORAGE_DIR", "storage_dir")
if not STORAGE_DIR.startswith('/'):
    STORAGE_DIR = os.path.join('../..', STORAGE_DIR)  # We are in local/loader
```
I think this solves the rel/abs path problem right? 
It didn't work as intended if `STORAGE_DIR` was declared, but as a relative path.